src/main/java/wdl4s/parser/MemoryUnit.java

```diff
@@ -0,0 +1,22 @@
+package wdl4s.parser;
+
+public enum MemoryUnit {
```

Comment:
I *think* so?  There was an enum here prior to this called `MemorySize`, so I just kept with the same paradigm of defining the enum in Java.